src/actions/admin.js

```diff
@@ -547,3 +547,23 @@ export function disablePlugin(pluginId: string): ActionFunc {
         return {data: true};
     };
 }
+
+export function getOAuthAppInfo(clientId: String): ActionFunc {
+    return bindClientFunc({
+        clientFunc: Client4.getOAuthAppInfo,
+        params: [clientId],
+    });
+}
+
+export function allowOAuth2(params: URLSearchParams): ActionFunc {
```

Comment:
Minor nit, but it would be nice if we could just use a plain object for the params instead of a `URLSearchParams` since I could see this possibly being used where those values aren't coming directly from the URL. That's just a thought and not something we have to change though.

Reply:
Yes nice point, making it plain object would broaden it's usage